tests/accuracy/test_global_correction.py

```diff
@@ -157,6 +159,54 @@ def test_lhc_global_correct(tmp_path, model_inj_beams, orientation):
         diff_rms_prev = diff_rms
 
 
+@pytest.mark.basic
+@pytest.mark.parametrize('dpp', (-2e-4, -1e-4, 1e-4, 7.5e-4))
```

Comment:
Are the runs different for these different dpps? If not, I would simply test one (or maybe with +-), to reduce the total testing time. Otherwise it's just like running the same test 4 times.